scalyr_agent/builtin_monitors/kubernetes_openmetrics_monitor.py

```diff
@@ -539,6 +541,18 @@ def _initialize(self):
 
         self.__previous_running_monitors_count = 0
 
+        self.__enable_monitor = self._global_config.k8s_explorer_enable
+
+    def run(self):
+        if not self.__enable_monitor:
```

Comment:
I guess that shouldn't be a big deal, unless there are some periodic "health checks" for running monitors, but I don't think that we have such checks now.

Reply:
Yeah, correct - and we already use the same pattern in the kubernetes_events_monitor (which I copied).

Reply:
Technically, we could also utilize "dynamic monitor scheduling functionality" which I implemented recently (monitor itself would ask MonitorsManagers to remove itself from running monitors), but this would be introducing a new pattern (and possible edge cases), so I went with the existing approach.